TheOracle2/AskTheOracle/OracleAnswer.cs

```diff
@@ -0,0 +1,63 @@
+namespace TheOracle2.GameObjects;
+
+public class OracleAnswer : Die, IMatchable
+{
+  public OracleAnswer(Random random, int odds, string question) : base(random, 100)
+  {
+    if (odds is < 1 or > 99) { throw new ArgumentOutOfRangeException(nameof(odds), "Chance must be from 1 to 99, inclusive."); }
+    Question = question;
+    Odds = odds;
+  }
+  public string Question { get; set; }
+  public int Odds { get; }
+  public bool IsYes => Value <= Odds;
+  public bool IsMatch => MatchResults.Contains(Value);
+  private static readonly List<int> MatchResults = new() { 11, 22, 33, 44, 55, 66, 77, 88, 99, 100 };
+
+  public override string ToString()
+  {
+    return $"Rolled {Value} vs. {Odds}: **{AnswerString}**";
+  }
+  public Color OutcomeColor()
+  {
+    return IsYes switch
+    {
+      true => new Color(0x47AEDD),
+      false => new Color(0xC50933)
+    };
+  }
+  private static string MatchMessage => "You rolled a match! Envision an extreme result or twist.";
+  private string AnswerString
+  {
+    get
+    {
+      string str = IsYes ? "Yes" : "No";
+      if (IsMatch)
+      {
+        str = $"hell {str}!".ToUpperInvariant();
```

Comment:
> Does this need to be changed to something else?

<img width="371" alt="Screen Shot 2022-01-17 at 8 54 07 PM" src="https://user-images.githubusercontent.com/5354757/149873179-8d72d3c6-b858-4728-845e-0541737e8b90.png">

hey, it's canon :D seriously though, yeah, probably makes more sense to have it as 2-4 localizable strings rather than interpolating it

Reply:
So it is! We can wait for localization (if we ever get there).